patches/minecraft/net/minecraft/block/BlockRedstoneComparator.java.patch

```diff
@@ -7,7 +7,7 @@
 +    @Override
 +    public void onNeighborChange(IBlockAccess world, BlockPos pos, BlockPos neighbor)
 +    {
-+        if (pos.func_177956_o() == neighbor.func_177956_o() && world instanceof World)
++        if (pos.func_177956_o() == neighbor.func_177956_o() && world instanceof net.minecraft.world.WorldServer)
```

Comment:
@diesieben07 It *may* be a bit more idiomatic to test `!((World)world).isRemote` after `world instanceof World`, but checking for `net.minecraft.world.WorldServer` (as you have done) *should* be equivalent, due to the way Minecraft works.